extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -129,6 +130,9 @@ object PrivilegesBuilder {
         val db = quote(parts.init)
         privilegeObjects += tablePrivileges(TableIdentifier(parts.last, Some(db)))
 
+      case permanentViewMarker: PermanentViewMarker =>
```

Comment:
case PermanentViewMarker(child, table) => mergeProjection(table, child)

looks we need to change to the above one, we may still project a view